### DISCUSSION TITLE
How can I set CollapsedSize to absolute width?

### DISCUSSION THREAD

**Discussion by kobe651jp** [bvaughn/react-resizable-panels#355] [Q&A] [answered]
For example, if you set the value to 2.5, in 1920 screen would be 48, and half size would be half the width.

Is there a way to set an absolute width? Like 3rem or 48px.

**Comment by bvaughn** [comment]
Pixel-based size constraints are not supported. The [FAQ covers this](https://github.com/bvaughn/react-resizable-panels?tab=readme-ov-file#can-panel-sizes-be-specified-in-pixels):

> ### Can panel sizes be specified in pixels?
> 
> No. Pixel-based constraints [added significant complexity](https://github.com/bvaughn/react-resizable-panels/pull/176) to the initialization and validation logic and so I've decided not to support them. You may be able to implement a version of this yourself following [a pattern like this](https://github.com/bvaughn/react-resizable-panels/issues/46#issuecomment-1368108416) but it is not officially supported by this library.

**Reply by bvaughn** [reply, marked as answer]
FYI pixel based constraints are supported in the latest version of the library.
